Makefile

```diff
@@ -54,6 +54,10 @@ ifneq '' '$(or $(filter clean,$(MAKECMDGOALS)),$(LLAMA_METAL))'
 BUILD_TARGETS += metal
 endif
 
+ifeq ($(UNAME_S),Darwin)
+BUILD_TARGETS += swift
+endif
```

Comment:
I afraid it affect some another user who use `make` with old versions of build tools, so probably need make it as a option, and recommend use `make swift` instead. (need update CI)

Reply:
✅